awx/ui/src/screens/Instances/InstanceDetail/InstanceDetail.js

```diff
@@ -127,6 +140,11 @@ function InstanceDetail({ setBreadcrumb }) {
     debounceUpdateInstance({ capacity_adjustment: roundedValue });
   };
 
+  const buildLinkURL = (inst) =>
+    inst.is_container_group
+      ? '/instance_groups/container_group/'
```

Comment:
TIL about container groups. I should probably make the same changes to my corresponding PR.